playbooks/byo/openshift-cluster/upgrades/docker/upgrade.yml

```diff
@@ -0,0 +1,137 @@
+# Playbook to upgrade Docker to the max allowable version for an OpenShift cluster.
+#
+- hosts: localhost
+  connection: local
+  become: no
+  gather_facts: no
+  tasks:
+  - include_vars: ../../../../../byo/openshift-cluster/cluster_hosts.yml
+  - add_host:
+      name: "{{ item }}"
+      groups: l_oo_all_hosts
+    with_items: g_all_hosts | default([])
```

Comment:
One thing I just noticed yesterday (and haven't taken the time to see if it occurs with ansible-1.9.x, but these add_hosts tasks show as 'changed', we should probably add `changed_when: false` to these to avoid that.